Source/Core/DolphinWX/TASInputDlg.h

```diff
@@ -33,10 +39,14 @@ class TASInputDlg : public wxDialog
   void CreateWiiLayout(int num);
 
 private:
-  static constexpr int ID_C_STICK = 1001;
-  static constexpr int ID_MAIN_STICK = 1002;
-  static constexpr int ID_CC_L_STICK = 1003;
-  static constexpr int ID_CC_R_STICK = 1004;
+  enum : int
+  {
+    ID_C_STICK = 1001,
+    ID_MAIN_STICK = 1002,
+    ID_CC_L_STICK = 1003,
+    ID_CC_R_STICK = 1004
+  };
+
   int m_eleID = 1005;
```

Comment:
> Does m_eleID have anything to do with the enum above? If so...

The ones in the enum are stick IDs. The mutable one is for various other UI elements that are created on the fly depending on layout. 1005 is just the next available ID that wouldn't clash with previous ones in the event system.

>  Is m_eleID compliant to the current coding style

No, `m_ele_id` would be, though. I don't like mixing large (i.e. if I make this one conform, I'd go through and do all the others as well) stylistic changes with functional/interface changes.

>